_maps/WIP/Sulaco_bottom.dmm

```diff
@@ -920,20 +919,18 @@
 	},
 /area/sulaco/hangar/one)
 "cm" = (
-/obj/machinery/power/apc{
-	dir = 8;
-	name = "Morgue Maintenance";
-	pixel_x = -30;
-	pixel_y = 0
-	},
 /obj/structure/cable{
 	d2 = 2;
 	icon_state = "0-2";
 	pixel_y = 0
 	},
 /obj/structure/cable{
-	icon_state = "0-4";
-	d2 = 4
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4"
```

Comment:
Same

Reply:
This is necessary so you can actually see how the powernet is wired while mapping.